bot/commands.py

```
@@ -1,300 +1,457 @@
+            return int(value)
+        elif key in ["TV_COLOR", "MOVIE_COLOR", "ANNOTATION_COLOR"]:
+            return format_color_value(value)
+        elif key == "FIXED_UPDATE_TIME":
+            return self._validate_fixed_update_time(value)
+        return value
+
+    def _validate_fixed_update_time(self, value: str):
+        try:
+            return self.validate_fixed_update_time(value)
+        except ValueError:
+            raise ValueError(self.translations["error_invalid_fixed_time"])
```

Ruff

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)